lib/features/steps/request_assertion_steps.rb

```diff
@@ -25,7 +25,8 @@
     received = (Server.stored_requests.size >= request_count)
     sleep 0.1
   end
-  raise "Requests not received in 30s (received #{Server.stored_requests.size})" unless received
+  raise "Expected #{request_count} requests but received #{Server.stored_requests.size} within the 30s timeout. This could indicate that Bugsnag crashed with a fatal error, or that it hasn’t made the requests that it should have done. Please check the device logs to confirm." unless received
```

Comment:
To eliminate Rubocop violations:
```suggestion
  unless received
    raise "Expected #{request_count} requests but received #{Server.stored_requests.size} within the 30s timeout. " \
      'This could indicate that Bugsnag crashed with a fatal error, or that it hasn’t made the requests that it ' \
      'should have done. Please check the device logs to confirm.'
  end
```